_config.yml

```diff
@@ -3,7 +3,8 @@ description: 'Open Source API Design and Registry.'
 url: 'https://www.apicur.io'
 _baseurl: 'https://www.apicur.io'
 github_username: apicurio
-github_url: https://github.com/apicurio
+github_linkedin: https://www.linkedin.com/company/djai
```

Comment:
this property name shouldn't be `github_linkedin` as LinkedIn is independent of GitHub at all.
Please name it: `linkedin_url` instead.
@AndreiZiatiev

Reply:
done